test/makeRequest-test.js

```diff
@@ -3,6 +3,7 @@ import request from 'request-promise-native';
 import makeRequest from '../src/makeRequest';
 
 jest.mock('request-promise-native');
+jest.setTimeout(30000);
```

Comment:
Here too, we should probably use jest mocking to prevent real requests from being made instead of increasing the timeout.

Reply:
The timeout is because async-retry uses a slightly longer timeout by default (also, we were already on the edge of the previous timeout).

Reply:
Oh I see. In that case, it seems like we can fix this and make this test faster by using jest's fake timers. https://jestjs.io/docs/en/timer-mocks

Reply:
Sure, although testing performance isn't really a huge problem in this repo.